examples/writing-filters/01-input.conf

```diff
@@ -0,0 +1,23 @@
+input {
+  generator {
+
+    # Add your example loglines here. It's a list, so you can add as many as you like.
```

Comment:
Maybe add a comment about how certain characters that may appear in log lines need to be escaped.